src/components/layout/Footer.vue

```diff
@@ -3,12 +3,19 @@
     v-spacer
       | © {{ new Date().getFullYear() }} - Made with 
       b.red--text ❤ 
-      | by QA
+      | by 
+      a(href="https://www.wolox.com.ar/")
+        | WOLOX
 </template>
 
 <style lang="scss" scoped>
 .footer {
-  background: none;
+  background-color: #363636;
+  color: whitesmoke;
+  a {
```

Comment:
Don't style HTML tags, it's not a good practice. What you need to do is to add a class to that element and then style that class